.env.example

```
@@ -23,4 +23,16 @@ UPLOADTHING_TOKEN="uplt tk"
+
+# Azure OpenAI — assistente de chat (intranet)
+AZURE_OPENAI_API_KEY="sua-chave"
+# URL do recurso (portal). O app normaliza para .../openai para o AI SDK.
+AZURE_OPENAI_ENDPOINT="https://meu-recurso.openai.azure.com"
+# Se preferir expor o endpoint Cognitive como variável separada, pode usar só esta (ou como fallback quando OPENAI_ENDPOINT não existir):
+# AZURE_COGNITIVE_ENDPOINT="https://meu-recurso.openai.azure.com"
+AZURE_OPENAI_DEPLOYMENT_NAME="gpt-4o"
+AZURE_OPENAI_API_VERSION="2024-10-21"
+# Avançado: host ou prefixo (o app normaliza para .../openai/deployments, exigido pelo AI SDK Azure v1)
+# AZURE_OPENAI_BASE_URL="https://meu-recurso.openai.azure.com"
+# AZURE_RESOURCE_NAME="meu-recurso"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)